### PR TITLE
AnneJu step 9-2: Set TimeZone and Error page

### PR DESCRIPTION
**Pull request by anneju**
1. Set up timezone to be `Tokyo`
2. Add customized error page.

- Related RSpecs
```
rspec spec/system/tasks_spec.rb:92:158
```

### CODE REVIEW

File: myapp/app/views/errors/404.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>The page you were looking for doesn't exist (404)</title>
```
Comment (anneju) [inline myapp/app/views/errors/404.html:5]: I have no taste in designing any pages 🙈 probably just leave it `simple style`

File: myapp/app/views/errors/404.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<h1><%= I18n.t('errors.404.title').capitalize %></h1>
+<p><%= I18n.t('errors.404.content').capitalize %></p>
```
Comment (anneju) [inline myapp/app/views/errors/404.html.erb:2]: I have no taste in designing any pages 🙈 probably just leave it `simple style`

File: myapp/config/locales/errors/en.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+en:
+  errors:
+    404:
+      title: The page you were looking for doesn't exist.
+      content: You may have mistyped the address or the page may have moved.
+    500:
+      title: We're sorry, but something went wrong. QQ
```
Comment (yo-it-engineer) [inline myapp/config/locales/errors/en.yml:7]: Q.Q

**Review by yo-it-engineer — APPROVED** [review]
LGTM！

Image from iOS (15)

File: myapp/config/locales/errors/en.yml (hunk 3 above)
Comment (anneju) [reply myapp/config/locales/errors/en.yml:7]: QQ

**Pull request merged**